examples/0_basic/6_create_vp.rs

```diff
@@ -174,46 +177,58 @@ async fn main() -> anyhow::Result<()> {
-
-  let presentation_validation_options = JwtPresentationValidationOptions::default()
-    .presentation_verifier_options(presentation_verifier_options)
-    .shared_validation_options(credential_validation_options)
-    .subject_holder_relationship(SubjectHolderRelationship::AlwaysSubject);
-
-  let (holder_did, issuer_dids): (CoreDID, Vec<CoreDID>) = JwtPresentationValidator::extract_dids(&presentation_jwt)?;
-
-  // Resolve issuer and holder documents and verify presentation.
-  // Passing the holder and issuer to `verify_presentation` will bypass the resolution step.
   let mut resolver: Resolver<IotaDocument> = Resolver::new();
   resolver.attach_iota_handler(client);
 
+  // Resolve the holder's document.
+  let holder_did: CoreDID = JwtPresentationValidator::extract_holder(&presentation_jwt)?;
   let holder: IotaDocument = resolver.resolve(&holder_did).await?;
-  let issuer: IotaDocument = resolver.resolve(&issuer_dids[0]).await?;
-
-  let presentation: DecodedJwtPresentation = JwtPresentationValidator::new().validate(
-    &presentation_jwt,
-    &holder,
-    &[issuer],
-    &presentation_validation_options,
-    FailFast::FirstError,
-  )?;
 
+  // Validate presentation. Note that this doesn't validate the included credentials.
+  let presentation_validation_options =
+    JwtPresentationValidationOptions::default().presentation_verifier_options(presentation_verifier_options);
+  let presentation: DecodedJwtPresentation =
+    JwtPresentationValidator::new().validate(&presentation_jwt, &holder, &presentation_validation_options)?;
+
+  // Concurrently resolve the issuers' documents.
+  let jwt_credentials = &presentation.presentation.verifiable_credential;
+  let issuers: Vec<CoreDID> = jwt_credentials
+    .iter()
+    .map(CredentialValidator::extract_issuer_from_jwt)
+    .collect::<Result<Vec<CoreDID>, _>>()?;
+  let issuers_documents: HashMap<CoreDID, IotaDocument> = resolver.resolve_multiple(&issuers).await?;
+
+  // Validate the credentials in the presentation.
+  let credential_validator: CredentialValidator = CredentialValidator::new();
+  let validation_options = CredentialValidationOptions::default();
+  for (index, jwt_vc) in jwt_credentials.iter().enumerate() {
+    //SAFETY: Indexing should be fine since we extracted the DID from each credential.
+    let issuer_document = issuers_documents
+      .get(&issuers[index])
+      .ok_or_else(|| anyhow::anyhow!("expected DID to be resolved"))?;
+    let decoded_credential: DecodedJwtCredential<Object> = credential_validator
+      .validate::<_, Object>(jwt_vc, issuer_document, &validation_options, FailFast::FirstError)
+      .unwrap();
+
+    // Check that the presentation holder matches the credential subject.
+    CredentialValidator::check_subject_holder_relationship(
+      &decoded_credential.credential,
+      &holder_did.to_url().into(),
+      SubjectHolderRelationship::AlwaysSubject,
+    )?;
+  }
```

Comment:
```suggestion
  }

```

Nit: Add a newline here so this section looks less crowded :)